I get the following warnings after the latest changes:
```
Marlin\src\feature\pause.cpp: In function 'void resume_print(const float&, const float&, const float&, int8_t, int16_t)':
Marlin\src\feature\pause.cpp:595:3: warning: this 'if' clause does not guard... [-Wmisleading-indentation]
   if (targetTemp > thermalManager.degTargetHotend(active_extruder))
   ^~
Marlin\src\feature\pause.cpp:599:5: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the 'if'
     load_filament(slow_load_length, fast_load_length, purge_length, max_beep_count, true, nozzle_timed_out, PAUSE_MODE_SAME DXC_PASS);
```
`this 'if' clause does not guard... [-Wmisleading-indentation]`

--- a/Marlin/src/feature/pause.cpp
+++ b/Marlin/src/feature/pause.cpp
@@ -592,8 +592,9 @@ void resume_print(const float &slow_load_length/*=0*/, const float &fast_load_le
     thermalManager.reset_hotend_idle_timer(e);
   }
 
-  if (targetTemp > thermalManager.degTargetHotend(active_extruder))
+  if (targetTemp > thermalManager.degTargetHotend(active_extruder)) {
     thermalManager.setTargetHotend(targetTemp, active_extruder);
+  }
 
   // Load the new filament
   load_filament(slow_load_length, fast_load_length, purge_length, max_beep_count, true, nozzle_timed_out, PAUSE_MODE_SAME DXC_PASS);

